include/net/nrf_cloud.rst

```diff
@@ -22,64 +25,44 @@ Before using any other APIs of the module, the application must call :cpp:func:`
+If the API fails, the application must retry to connect.
 
-
-.. code-block:: c
-
-   /* Array of supported user association methods. */
-   const enum nrf_cloud_ua supported_uas[] = {
-		NRF_CLOUD_UA_BUTTON
-   };
-
-   /* Array of supported sensors. */
-   const enum nrf_cloud_sensor supported_sensors[] = {
-		NRF_CLOUD_SENSOR_GPS,
-		NRF_CLOUD_SENSOR_FLIP
-   };
-
-   /* List of supported user association methods. */
-   const struct nrf_cloud_ua_list ua_list = {
-		.size = ARRAY_SIZE(supported_uas),
-		.ptr = supported_uas
-   };
-
-   /* List of supported sensors. */
-   const struct nrf_cloud_sensor_list sensor_list = {
-		.size = ARRAY_SIZE(supported_sensors),
-		.ptr = supported_sensors
-   };
-
-   /* Struct containing both lists. */
-   const struct nrf_cloud_connect_param param = {
-		.ua = &ua_list,
-		.sensor = &sensor_list,
-   };
-
-   int err = nrf_cloud_connect(&param);
-
-
-First, the library tries to establish the transport for communicating with the cloud. This procedure involves a TLS handshake that might take up to three seconds. The API blocks for the duration of the handshake.
+First, the library tries to establish the transport for communicating with the cloud.
+This procedure involves a TLS handshake that might take up to three seconds.
+The API blocks for the duration of the handshake.
 
 Next, the API subscribes to an MQTT topic to start receiving user association requests from the cloud.
+The cloud uses the device's certificates for authentication.
+The certificates are generated using the device's ID and PIN/HWID.
 
-Every time nRF Cloud starts a communication session with a device, it verifies if the device is uniquely associated with a user. If not, the user association procedure is triggered. The following message sequence chart shows the flow of events and expected application responses to each event:
+Every time nRF Cloud starts a communication session with a device, it verifies whether the device is uniquely associated with a user.
+If not, the user association procedure is triggered.
+When adding the device to an nRF cloud account, the user must provide the correct device ID and PIN/HWID to nRF Cloud.
+
+The following message sequence chart shows the flow of events and expected application responses to each event during the user association procedure:
 
 .. msc::
    hscale = "1.3";
    Module,Application;
    Module<<Application      [label="nrf_cloud_connect() returns successfully"];
    Module>>Application      [label="NRF_CLOUD_EVT_TRANSPORT_CONNECTED"];
    Module>>Application      [label="NRF_CLOUD_EVT_USER_ASSOCIATION_REQUEST"];
-   Module<<Application      [label="nrf_cloud_user_associate()"];
+    ---                     [label="Add the device to nRF Cloud account"];
    Module>>Application      [label="NRF_CLOUD_EVT_USER_ASSOCIATED"];
-   Module>>Application      [label="NRF_CLOUD_EVT_READY"];
+   Module<<Application      [label="Application calls nrf_cloud_disconnect()"];
```

Comment:
Above we say "nrf_cloud_connect() returns successfully". Is it important to stress that it's the application calling the function here? This should be clear from the direction.

Reply:
I think the words are more helpful than the `<<`.
I can remove the words if you prefer.

Reply:
Have you checked the output?
I don't mind either way, but it should be consistent (either always mentioning the direction or never mentioning it).